Source/WebInspectorUI/UserInterface/Views/ObjectTreeView.css

```diff
@@ -84,7 +84,7 @@
 
 .tree-outline.object {
     margin: 0;
-    padding: 0 6px 2px;
+    padding: 0 0 2px 6px;
```

Comment:
i wanna say that even though the Console is always LTR, there are places where object trees are shown RTL (e.g. hovering over a token when JavaScript is paused the Sources Tab)

if that's the case, do these still work when in RTL?

Reply:
What specific test case do you have in mind, like what's an instance of a RTL object tree? The popover with the object tree looks fine to me in the Sources tab. Sure, it's without the 6px right padding when scrolled all the way to the right, but the difference is then when the tree is nested, there won't be a large stack of right padding applied on the right. Given there's no left padding, a right padding shouldn't be necessary.

Reply:
i dont have a specific situation in mind, more of trying to follow the general principle of having the UI look exactly the same when in RTL as it does when in LTR

> it's without the 6px right padding when scrolled all the way to the right

this makes me think that maybe we should have the CSS be something like this instead?

```css
padding-top: 0;
padding-inline: 6px 0;
padding-bottom: 2px;
```

put another way, whenever i see `padding-left`/`padding-right` i often think "should this be `padding-inline-start`/`padding-inline-end` instead?"

Reply:
Ah, I see what you mean. Let's use `padding-inline` instead and avoid having to worry about the `left` and `right` ness.